app/gui/src/ide/integration.rs

```diff
@@ -38,15 +38,20 @@ impl Model {
         if let Some(project_model) = self.controller.current_project() {
             // We know the name of new project before it loads. We set it right now to avoid
             // displaying placeholder on the scene during loading.
-            self.view.graph().model.breadcrumbs.project_name(project_model.name().to_string());
+            self.view
+                .project_view()
+                .graph()
+                .model
+                .breadcrumbs
+                .project_name(project_model.name().to_string());
 
             let status_notifications = self.controller.status_notifications().clone_ref();
             let project = controller::Project::new(project_model, status_notifications.clone_ref());
 
             executor::global::spawn(async move {
                 match project.initialize().await {
                     Ok(result) => {
-                        let view = self.view.clone_ref();
+                        let view = self.view.project_view().clone_ref();
```

Comment:
The naming seems strange here. `view. ...` suggests that we are getting to a "view", and then we are repeating the "view" name again. I'd understand `self.view.project`, or `self.project_view`, but `self.view.project_view` seems like a badly refactored names.

Reply:
Renamed it to `project()`